backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/PrototypeCleanupServiceTest.java

```diff
@@ -0,0 +1,178 @@
+
+    underTest.clean(norm);
+    assertThat(
+      norm.getRegelungstext1().getMeta().getProprietary().get().getMetadataValue(Metadata.SUBTYP)
+    )
+      .isEmpty();
+    assertThat(
+      norm
+        .getRegelungstext1()
+        .getMeta()
+        .getProprietary()
+        .get()
+        .getMetadataValue(Metadata.BEZEICHNUNG_IN_VORLAGE)
+    )
+      .isEmpty();
+    assertThat(
+      norm
+        .getRegelungstext1()
+        .getMeta()
+        .getProprietary()
+        .get()
+        .getMetadataValue(Metadata.ART_DER_NORM)
+    )
+      .isEmpty();
+    assertThat(
+      norm.getRegelungstext1().getMeta().getProprietary().get().getMetadataValue(Metadata.NORMGEBER)
+    )
+      .isEmpty();
+    assertThat(
+      norm.getRegelungstext1().getMeta().getProprietary().get().getMetadataValue(Metadata.SUBTYP)
+    )
+      .isEmpty();
+    assertThat(
+      norm
+        .getRegelungstext1()
+        .getMeta()
+        .getProprietary()
+        .get()
+        .getMetadataValue(Metadata.BEZEICHNUNG_IN_VORLAGE)
+    )
+      .isEmpty();
+    assertThat(
+      norm.getRegelungstext1().getMeta().getProprietary().get().getMetadataValue(Metadata.NORMGEBER)
+    )
+      .isEmpty();
+    assertThat(
+      norm
+        .getRegelungstext1()
+        .getMeta()
+        .getProprietary()
+        .get()
+        .getMetadataValue(Metadata.BESCHLIESSENDES_ORGAN)
+    )
+      .isEmpty();
+    assertThat(
+      norm
+        .getRegelungstext1()
+        .getMeta()
+        .getProprietary()
+        .get()
+        .getMetadataValue(Metadata.ORGANISATIONS_EINHEIT)
+    )
+      .isEmpty();
+    assertThat(
+      norm
+        .getRegelungstext1()
+        .getMeta()
+        .getProprietary()
+        .get()
+        .getMetadataValue(Metadata.ENTRY_INTO_FORCE_DATE)
+    )
+      .contains("2008-07-01");
+    assertThat(
+      norm
+        .getRegelungstext1()
+        .getMeta()
+        .getProprietary()
+        .get()
+        .getMetadataValue(Metadata.EXPIRY_DATE)
+    )
+      .contains("2011-12-31");
+    assertThat(
+      norm
+        .getRegelungstext1()
+        .getMeta()
+        .getProprietary()
+        .get()
+        .getMetadataValue(Metadata.STANDANGABE)
+    )
+      .contains(
+        "V aufgeh. durch Art. 22 G v. 22.12.2011 I 1111 mWv\n                       1.1.2012"
+      );
+  }
+
+  @Test
+  void cleanRegularAndBundesregierungMetadata() {
+    final Norm norm = Fixtures.loadNormFromDisk("NormWithProprietaryToBeCleaned.xml");
+
+    underTest.clean(norm);
+    List<Node> proprietyChildNodes = NodeParser
+      .nodeListToList(
+        norm.getRegelungstext1().getMeta().getProprietary().get().getElement().getChildNodes()
+      )
+      .stream()
+      .filter(node -> node.getNodeType() == Node.ELEMENT_NODE)
+      .toList();
+
+    assertThat(proprietyChildNodes).hasSize(1);
+    assertThat(proprietyChildNodes.get(0).getNamespaceURI())
+      .isEqualTo("http://MetadatenRIS.LegalDocML.de/1.7.2/");
```

Comment:
i would use `Namespace.METADATEN_RIS.getNamespaceUri()` so we don't need to touch this when updating the ldml version